integration/next_search_test.go

```diff
@@ -13,12 +13,13 @@ func TestNextSearchWord(t *testing.T) {
 
 	program := testCmd(repo)
 
-	output, exiterr, success := program("add", "one")
+	output, exiterr, success := program("add", "one", "/", "alpha")
```

Comment:
add a string to the Note field here

Reply:
So searching notes is a separate thing? Right now searching with dstask searches the description and note implicitly. I think this works well given the fairly limited scope of searching non-resolved tasks by default. What do you think?

Reply:
Here we simply add a note to task one, but below we do check for
* searching for something that isn't there (no results)
* finding string "alpha" in a note
* finding task "two" via its summary

Is there another test we should add? The strings are admittedly really simple.

Reply:
Sorry I misread, late at night. Seems fine as is.